Add how to supply ext args with params (#2832)
nf-core/website: sites/docs/src/content/docs/guidelines/components/modules.md

file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
```

### Configuration of ext.args in tests

> Module nf-tests SHOULD use a single `nextflow.config` to supply `ext.args` to a module, by taking the settings from the
> params block defined in the `when` block of a test.

Reviewer: To me the sentence was difficult to read that is why I tried to change it a little
```suggestion
Module nf-tests SHOULD use a single `nextflow.config` to supply `ext.args` to a module. They can be defined in the `when` block of a test under the `params` scope.
```

```groovy title="main.nf.test"
config './nextflow.config'

when {
  params {
    module_args = '--extra_opt1 --extra_opt2'
  }
  process {
    """
    input[0] = [
      [ id:'test1', single_end:false ], // meta map
      file(params.modules_testdata_base_path + 'genomics/prokaryotes/bacteroides_fragilis/genome/genome.fna.gz', checkIfExists: true)
    ]
    """
  }
}
```

> ```groovy title="nextflow.config"

Author: I added something, although I don't think it's really necessary.

The syntax highlighting looked OK to me. Can you describe what's off?

> ```groovy title="nextflow.config"

Reviewer 2: I haven't checked the deploy preview admittedly, but in the github reviewing context, the normal text that should be white is blue like it thinks it's code:

![image](https://github.com/user-attachments/assets/3f8c1981-9ab5-4523-b082-fd71076d40d0)

> ```groovy title="nextflow.config"

Author: Ah, OK. I think I found it around line 184. I'll see if it fixes it.

> ```groovy title="nextflow.config"

Author: There's something I don't understand on Line 193 now that changes formatting.

> ```groovy title="nextflow.config"

Author: @mashehu Any idea why the highlighting goes funny on Line 196?
